index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name='viewport' content='width=device-width, initial-scale=1'>
+    <title>Max's Portfolio</title>
+    <link href='https://fonts.googleapis.com/css?family=Muli' rel='stylesheet' type='text/css'>
+    <link rel="stylesheet" href="vendor/normalize.css/normalize.css">
+    <link rel="stylesheet" href="styles/base.css">
+    <link rel="stylesheet" href="styles/layout.css">
+    <link rel="stylesheet" href="styles/modules.css">
+    <link rel="stylesheet" href="styles/icons.css">
+
+    <script id="project-template" type="text/x-handlebars-template">
+      <article class="clearfix" data-category="{{category}}" data-author="{{author}}">
+        <header>
+          <h2>{{title}}</h2>
+          <div class="authorInfo">
+              By <address><a href="{{authorUrl}}">{{author}}</a></address>
+              {{publishStatus}} <time pubdate datetime="2000-01-01">on {{publishedOn}}</time>
+          </div>
+        </header>
+        <section class="project-body">{{{body}}}</section>
+      </article>
+    </script>
+  </head>
+  <body>
+    <header class="main-header">
+      <h1>
+        Max Friedrichsen's Portfolio
+      </h1>
+      <p>Welcome!</p>
+    </header>
+    <nav class="head-nav">
+      <div class="icon-menu"></div>
+      <ul>
+        <li class="tab icon-home" data-content="projects"><a href="#">Home</a></li>
+        <li class="tab icon-profile" data-content="about"><a href="#">About</a></li>
+        <li class="icon-github"><a href='https://github.com/meeksfred' target='_blank'>GitHub</a></li>
+        <li class="icon-twitter"><a href='https://twitter.com/maxamillionfred' target='_blank'>Twitter</a></li>
+        <li class="icon-facebook2"><a href='https://www.facebook.com/max.friedrichsen' target='_blank'>Facebook</a></li>
+      </ul>
+    </nav>
+    <main>
+      <section id="projects" class="tab-content">
+        <div>
+          <ul id="filters">
+            <li>
+              <select id="category-filter">
+                <option value="">-- Categories --</option>
+              </select>
+            </li>
+          </ul>
+        </div>
+      </section>
+      <section id="about" class="tab-content">
```

Comment:
Make sure you aren't over-using tools when they aren't necessary.  Handlebars is a great way to add repeated sections to your page.  But if you are only going to have one about section, then there is not a reason to make it into a template.